docs/content/troubleshooting.md

```diff
@@ -1,113 +1,119 @@
+  * Ensure that your field is declared in a [MonoBehaviour]({{urlRoot}}/content/glossary#monobehaviour). They are not supported in ECS systems.
   * Ensure that the field is decorated with a [Require] attribute.
-  * Ensure that the GameObject containing your MonoBehaviour is associated with a SpatialOS entity. You can verify this by examining whether a `SpatialOSComponent` MonoBehaviour was added to your GameObject at runtime.
+  * Ensure that the GameObject containing your MonoBehaviour is associated with a SpatialOS entity. You can verify this by examining whether a [`SpatialOSComponent`]({{urlRoot}}/content/glossary#spatialos-component) MonoBehaviour is added to your GameObject at runtime.
   * Ensure that you only access Requirables in a MonoBehaviour while the MonoBehaviour is enabled. Requirables are supposed to be null while the MonoBehaviour is disabled. Note that certain Unity event methods like `OnTriggerEnter` or `OnCollisionEnter` may be invoked even if a MonoBehaviour is disabled.
 
-<br/>
-
-#### A MonoBehaviour containing fields using the [Require] attribute stays disabled when I expect it to be enabled
+#### A MonoBehaviour containing fields using the Require attribute remains disabled when I expect it to be enabled
 
-**Issue and workarounds**<br/>
+**Causes and fixes**<br/>
 
+This can be caused by multiple problems. Follow the checklist below to discover the cause and fix:
 
   * Ensure that `GameObjectRepresentationHelper.AddSystems` is called when initializing your [`Worker`]({{urlRoot}}/content/glossary#worker).
   * Ensure that the GameObject containing your MonoBehaviour is associated with a SpatialOS entity. You can verify this be examining whether a `SpatialOSComponent` MonoBehaviour was added to your GameObject at runtime.
-  * Ensure that the [requirements]({{urlRoot}}/content/gameobject/interact-spatialos-monobehaviours) for all your fields in your MonoBehaviour are satisfied. You can verify this by examining the SpatialOS entity associated with your GameObject in the Inspector. In the Inspector, ensure that all relevant components are attached to your SpatialOS entity and that read and write access is delegated correctly for your worker.
+  * Ensure that the [requirements]({{urlRoot}}/content/gameobject/interact-spatialos-monobehaviours) for all your fields in your MonoBehaviour are satisfied. You can verify this by examining the SpatialOS entity associated with your GameObject in the [Inspector]({{urlRoot}}/content/glossary#inspector). In the Inspector, ensure that all relevant components are attached to your SpatialOS entity and that read and write access is delegated correctly for your worker.
+
+#### When opening a Unity GDK solution in Visial Studio 2017, the file has no matching compilation unit
+
+**Cause**<br/>
+You don't have the correct compilation units installed.
+
+**Fix**<br/>
 
+1. Follow the Visual Studio instalation steps on our [setup page]({{urlRoot}}/setup-and-installing#set-up-your-machine)).
```

Comment:
Its _probably_ to update Visual Studio to latest, but I can't guarantee that for sure.